sources/Renderer/OpenGL/Platform/Linux/LinuxGLContextWayland.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * LinuxGLContext.h
+ *
+ * Copyright (c) 2025 Lukas Hermanns. All rights reserved.
+ * Licensed under the terms of the BSD 3-Clause license (see LICENSE.txt).
+ */
+
+
+#if LLGL_LINUX_ENABLE_WAYLAND
+
+#ifndef LLGL_LINUX_GL_CONTEXT_WAYLAND_H
+#define LLGL_LINUX_GL_CONTEXT_WAYLAND_H
+
+
+#include "LinuxGLContext.h"
+#include "../../OpenGL.h"
+#include <LLGL/RendererConfiguration.h>
+#include <LLGL/Platform/NativeHandle.h>
+#include <X11/Xlib.h>
+
+
+namespace LLGL
+{
+
+enum class LinuxGLAPIType : char {
```

Comment:
Is this enum still needed? I presume this was from your initial implementation before you split up the classes. Your `IsWayland()` function solves this more elegantly IMO.